<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerSubjectHierarchyModule</class>
 <widget class="qSlicerWidget" name="qSlicerSubjectHierarchyModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SubjectHierarchy</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <widget class="qMRMLSubjectHierarchyTreeView" name="SubjectHierarchyTreeView">
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragDrop</enum>
     </property>
     <property name="indentation">
      <number>12</number>
     </property>
     <property name="editMenuActionVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="horizontalSpacing">
      <number>4</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="1" column="0" colspan="2">
      <widget class="ctkCollapsibleGroupBox" name="MRMLNodeInspectorGroupBox">
       <property name="title">
        <string>MRML Node Inspector</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="margin">
         <number>4</number>
        </property>
        <property name="spacing">
         <number>4</number>
        </property>
        <item row="0" column="0">
         <widget class="qMRMLNodeAttributeTableWidget" name="MRMLNodeAttributeTableWidget"/>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="DisplayMRMLIDsCheckBox">
       <property name="text">
        <string>MRML ID's</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QCheckBox" name="DisplayTransformsCheckBox">
         <property name="text">
          <string>Transforms</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="PotentialNodesCheckBox">
         <property name="text">
          <string>Potential nodes</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_Help">
         <property name="toolTip">
          <string>Tooltip is populated by the plugins</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../qSlicerSubjectHierarchyModule.qrc">:/Icons/Help.png</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeAttributeTableWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeAttributeTableWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLTreeView</class>
   <extends>QTreeView</extends>
   <header>qMRMLTreeView.h</header>
   <slots>
    <slot>setSceneModel(int)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLSubjectHierarchyTreeView</class>
   <extends>qMRMLTreeView</extends>
   <header>qMRMLSubjectHierarchyTreeView.h</header>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>ctkCollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qSlicerSubjectHierarchyModule.qrc"/>
  <include location="../qSlicerSubjectHierarchyModule.qrc"/>
  <include location="../qSlicerSubjectHierarchyModule.qrc"/>
  <include location="../qSlicerSubjectHierarchyModule.qrc"/>
  <include location="../qSlicerSubjectHierarchyModule.qrc"/>
  <include location="../qSlicerSubjectHierarchyModule.qrc"/>
  <include location="../qSlicerSubjectHierarchyModule.qrc"/>
  <include location="../qSlicerSubjectHierarchyModule.qrc"/>
  <include location="../qSlicerSubjectHierarchyModule.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>qSlicerSubjectHierarchyModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>SubjectHierarchyTreeView</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>2</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
